### ISSUE TITLE
TypeError: Cannot read property 'hitCount' of undefined

### ISSUE THREAD

**Issue by petermr** [ContentMine/getpapers#84]
[I thought I had posted under this title before but can't find it]
```
localhost:projects pm286$ getpapers -q neonicotinoid -x -o neonicotinoid -k 100
info: Searching using eupmc API

TypeError: Cannot read property 'hitCount' of undefined
    at EuPmc.completeCallback (/Users/pm286/.nvm/v0.10.38/lib/node_modules/getpapers/lib/eupmc.js:64:11)
    at Request.emit (events.js:98:17)
    at Request.mixin._fireError (/Users/pm286/.nvm/v0.10.38/lib/node_modules/getpapers/node_modules/restler/lib/restler.js:207:10)
    at ClientRequest.<anonymous> (/Users/pm286/.nvm/v0.10.38/lib/node_modules/getpapers/node_modules/restler/lib/restler.js:246:14)
    at ClientRequest.emit (events.js:95:17)
    at Socket.socketErrorListener (http.js:1548:9)
    at Socket.emit (events.js:95:17)
    at net.js:834:16
    at process._tickCallback (node.js:448:13)
```
This was run when the internet connection had dropped. When I reconnected I didn't get it.

Can we test for connections by a method which gives a clean error and then drops the job?

**Comment by Blahah** [comment]
Should have been fixed in #77

**Comment by Blahah** [comment]
(and included in release `0.4.3`)
